### PR TITLE
Added Order Controller and all related stuff

### PR DESCRIPTION
**Pull request by Zel1oy**

### CODE REVIEW

**Review by AntonZhdanov — APPROVED** [review]

File: src/main/java/com/example/onlinebookstore/model/Order.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    @Enumerated(EnumType.STRING)
+    private Status status;
+    @Column(nullable = false)
+    private BigDecimal total;
+    @Column(nullable = false)
+    private LocalDateTime orderDate;
+    @Column(nullable = false)
+    private String shippingAddress;
+    @OneToMany(mappedBy = "order")
+    private Set<OrderItem> orderItems;
```
Comment (andrii-hoienko) [inline src/main/java/com/example/onlinebookstore/model/Order.java:49]: joinColumn? Maybe it makes sense to exclude it from equals/hashCode/toString?

File: src/main/java/com/example/onlinebookstore/service/impl/ShoppingCartServiceImpl.java [hunk 2]
```diff
@@ -55,8 +56,13 @@ public ShoppingCartDto addToCart(CreateCartItemRequestDto request) {
 
     @Override
     public CartItemDto updateCartItem(Long cartItemId, CartItemUpdateRequestDto request) {
-        CartItem cartItem = cartItemRepository.findById(cartItemId).orElseThrow(
-                () -> new EntityNotFoundException("Can't find cart item with id: " + cartItemId));
+        ShoppingCart shoppingCart = shoppingCartRepository.findByUserId(
+                userService.getAuthenticatedUser().getId())
+                .orElseThrow(() -> new EntityNotFoundException(
+                                "There is not cart belonging to current user"));
```
Comment (andrii-hoienko) [inline src/main/java/com/example/onlinebookstore/service/impl/ShoppingCartServiceImpl.java]: Add details to error message

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 3]
```diff
@@ -18,4 +18,8 @@ databaseChangeLog:
   - include:
       file: db/changelog/changes/09-create-shopping-carts-table.yaml
   - include:
-      file: db/changelog/changes/10-create-cart-items-table.yaml
\ No newline at end of file
+      file: db/changelog/changes/10-create-cart-items-table.yaml
```
Comment (andrii-hoienko) [inline src/main/resources/db/changelog/db.changelog-master.yaml:21]: add empty line

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
GJ, I left minor comments

File: src/main/java/com/example/onlinebookstore/model/Order.java (hunk 1 above)
Comment (Zel1oy) [reply src/main/java/com/example/onlinebookstore/model/Order.java:49]: I guess maybe i don't need joinColumn here because of mappedBy

**Review by andrii-hoienko — APPROVED** [review]
GJ!

**Pull request merged**